### PR TITLE
Update voice normalization rule

### PR DESCRIPTION
**Pull request by JensTrier**
Updating Get/Set/Remove to prepare for upcoming removal from TPM

Updating New/Test to be specific about the differences between on-premises usage and Microsoft Teams usage.

### CODE REVIEW

File: skype/skype-ps/skype/New-CsVoiceNormalizationRule.md [hunk 1]
```diff
@@ -37,13 +39,17 @@ New-CsVoiceNormalizationRule [-Tenant <Guid>] -Parent <String> -Name <String> [-
 ## DESCRIPTION
 This cmdlet creates a named voice normalization rule.
 These rules are a required part of phone authorization and call routing.
-They define the requirements for converting (or translating) numbers from an internal Skype for Business Server format to a standard (E.164) format.
+They define the requirements for converting (or translating) numbers from an internal format to a standard (E.164) format.
 An understanding of regular expressions is helpful in order to define number patterns that will be translated.
 
-Rules that are created by using this cmdlet are part of the dial plan and in addition to being accessible through the `Get-CsVoiceNormalizationRule` cmdlet, can also be accessed through the NormalizationRules property returned by a call to the `Get-CsDialPlan` cmdlet.
+For Lync or Skype for Business Server, rules that are created by using this cmdlet are part of the dial plan and in addition to being accessible through the 
+`Get-CsVoiceNormalizationRule` cmdlet, can also be accessed through the NormalizationRules property returned by a call to the `Get-CsDialPlan` cmdlet.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsVoiceNormalizationRule.md:46]:
```suggestion
The `Get-CsVoiceNormalizationRule` cmdlet can also be accessed through the NormalizationRules property returned by a call to the `Get-CsDialPlan` cmdlet.
```

File: skype/skype-ps/skype/New-CsVoiceNormalizationRule.md [hunk 2]
```diff
@@ -37,13 +39,17 @@ New-CsVoiceNormalizationRule [-Tenant <Guid>] -Parent <String> -Name <String> [-
...
 You cannot create a normalization rule unless a dial plan with an Identity matching the scope specified in the normalization rule Identity already exists.
 For example, you can't create a normalization rule with the Identity site:Redmond/RedmondNormalizationRule unless a dial plan for site:Redmond already exists.
 
+For Microsoft Teams,  rules that are created by using this cmdlet can only be created with the InMemory switch and should be added to a tenant dial plan using
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsVoiceNormalizationRule.md:50]:
```suggestion
For Microsoft Teams, rules that are created by using this cmdlet can only be created with the InMemory switch and should be added to a tenant dial plan using 
```

File: skype/skype-ps/skype/Test-CsVoiceNormalizationRule.md [hunk 3]
```diff
@@ -49,23 +49,49 @@ If this rule does not apply to the DialedNumber value (for example, if the norma
...
+Test-CsVoiceNormalizationRule -DialedNumber 1234 -NormalizationRule $nr[0]
+```
+```output
+TranslatedNumber
+----------------
++12065551234
+```
+
+For Microsoft Teams, this example gets all the normalization rules in the tenant dial plan DP1, shows the first of these rules and then test that rule on the
+dialed number 1234. The output shows that the rule normalize the dialed number to +12065551234
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Test-CsVoiceNormalizationRule.md:94]:
```suggestion
For Microsoft Teams, this example gets all the normalization rules in the tenant dial plan DP1, shows the first of these rules, and then test that rule on the
dialed number 1234. The output shows that the rule normalize the dialed number to +12065551234.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the few minor suggested changes. Many thanks.

**Comment by JensTrier** [comment]
> Copy review - @JensTrier, kindly commit the few minor suggested changes. Many thanks.

Thanks & Done!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd

**Pull request merged**